front/app/containers/Authentication/useSteps/stepConfig/sharedSteps.ts

```diff
@@ -113,7 +113,10 @@ export const sharedSteps = (
             return;
           }
 
-          setCurrentStep('success');
+          const { successAction } = getAuthenticationData();
+          if (successAction) {
+            triggerSuccessAction(successAction);
+          }
```

Comment:
Solves a small issue from the previous SSO work where the success step (which just triggers the action really) was showing unnecessarily after SSO sign in. We should be just triggering the action here :)